src/hind_brain/hind_brain.ino

```diff
@@ -106,10 +111,15 @@ void ackermannCB(const ackermann_msgs::AckermannDrive &drive) {
 
 } // ackermannCB()
 
+void watchdogCB(const std_msgs::Empty &msg) {
+  watchdog_timer = millis();
+} // watchdogCB()
+
 void checkSerial(ros::NodeHandle *nh) {
-  // Given node, estops if node is not connected
+  // Given node, estops if watchdog has timed out
   // https://answers.ros.org/question/124481/rosserial-arduino-how-to-check-on-device-if-in-sync-with-host/
-  if(!nh->connected()) {
+
+  if(millis() - watchdog_timer > WATCHDOG_TIMEOUT) {
```

Comment:
Here you use `>`, and on line 66 it seems you use `<`. What if `millis() - watchdog_timer = WATCHDOG_TIMEOUT`? I see no hazard, as the setup will pass and the estop will not trigger and going just 1 millisecond over `WATCHDOG_TIMEOUT` seems okay. However, semantically, this should be `>=` to enforce the semantics of the timeout.